pynest/nest/tests/test_aeif_lsodar.py

```diff
@@ -249,10 +249,10 @@ def assert_pass_tolerance(self, rel_diff, di_tol):
                                     model, var, diff, di_tol[model][var]))
 
     @unittest.skipIf(not HAVE_GSL, 'GSL is not available')
-    def test_closeness_nest_lsodar(self):
-        '''
-        Compare models to the LSODAR implementation.
-        '''
+    def testClosenessNestLSODAR(self):
+
+        # Compare models to the LSODAR implementation.
```

Comment:
If present, nostests takes the first line of the docstring for reporting on the test cases. This led to something like this:  `The models should behave as iaf_cond_* if a == 0., b == 0. and ... ok`
Most of the test cases make no use of a docstring. In that case the function name is reported. Hence I turned the docstring into line comments to make it clear and consistent.